pydough/sqlglot/transform_bindings/base_transform_bindings.py

```diff
@@ -343,6 +345,84 @@ def convert_replace(
             args.append(sqlglot_expressions.Literal.string(""))
         return sqlglot_expressions.Anonymous(this="REPLACE", expressions=args)
 
+    def convert_str_count(
+        self,
+        args: list[SQLGlotExpression],
+        types: list[PyDoughType],
+    ) -> SQLGlotExpression:
+        """Convert a `STRCOUNT` call expression to a SQLGlot expression. It counts how many times
+        the string Y appears in the string X.
+
+        STRCOUNT(X, Y) =>
+        CASE
+            WHEN LENGTH(Y) = 0 THEN 0
+            ELSE CAST((LENGTH(X) - LENGTH(REPLACE(X, Y, ''))) / LENGTH(Y), AS INTEGER)
```

Comment:
Same here with 80 line characters
```suggestion
            ELSE
            CAST((LENGTH(X) - LENGTH(REPLACE(X, Y, ''))) / LENGTH(Y), AS INTEGER)
```